openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -67,9 +68,88 @@ def _get_resname(off_mol) -> str:
     return names[0]
 
 
+def _get_alchemical_charge_difference(
+    mapping: LigandAtomMapping,
+    nonbonded_method: str,
+    explicit_charge_correction: bool,
+    positive_ion_resname: str = 'NA',
+    negative_ion_resname: str = 'CL',
+) -> int:
+    """
+    Checks and returns the difference in formal charge between
```

Comment:
these first lines in docstrings are meant to be a one liner.  So `Difference in formal charge between state A and B` gives the same information

Reply:
done